src/internal/gateways/genius/action-validate-data.ts

```diff
@@ -0,0 +1,29 @@
+import { IErrorReason } from "..";
+import { IDictionary } from "../../lib/util";
+
+export default (data: IDictionary) => {
+  const errors: IErrorReason[] = [];
+
+  if (!data["card-number"] && !data["card-track"] && !data["account-number"]) {
```

Comment:
Should there be an error for no card-track thrown in here somewhere or some sort of default error thrown?

Reply:
Ah. The track and eCheck checks should probably be removed. It's unknown if Genius' tokenization service supports those.

Reply:
Updated in 890a166f9fd1117bf7cb86034efa487cbe14a656